.circleci/config.yml

```diff
@@ -368,78 +368,78 @@ jobs:
 workflows:
   openlineage:
     jobs:
-      - build-client-java
-      - publish-snapshot-client-java:
-          <<: *only_on_main
-          context: release
-          requires:
-            - build-client-java
-      - build-integration-spark:
-          matrix:
-            parameters:
-              spark-version: [ '2.4.1', '3.1.2' ]
+      # - build-client-java
+      # - publish-snapshot-client-java:
+      #     <<: *only_on_main
+      #     context: release
+      #     requires:
+      #       - build-client-java
+      # - build-integration-spark:
+      #     matrix:
+      #       parameters:
+      #         spark-version: [ '2.4.1', '3.1.2' ]
```

Comment:
Accidental checkin?

Reply:
You managed to look there at the time when I was fixing integration tests 😉